dev-cpp/robin-map/robin-map-9999.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="C++ fast hash map and hash set using robin hood hashing"
```

Comment:
Do we need a live ebuild for this?

Reply:
I think it is fine to have it but not necessary.